zerver/lib/message.py

```diff
@@ -78,6 +78,7 @@ def to_dict_uncached_helper(message, apply_markdown):
                 recipient_id = message.recipient.id,
                 recipient_type = message.recipient.type,
                 recipient_type_id = message.recipient.type_id,
+                reactions = message.reaction_set.all()
```

Comment:
`reaction_set` is wrong method here, I am replacing it with `Reaction.get_raw_db_rows([message.id]`